content/v2.0/reference/cli/influx/config/create.md

```diff
@@ -0,0 +1,27 @@
+---
+title: influx config create
+description: The 'influx config create' command creates a new InfluxDB connection configuration.
+menu:
+  v2_0_ref:
+    name: influx config create
+    parent: influx config
+weight: 201
+---
+
+The `influx config create` command creates a new InfluxDB connection configuration and stores it in the `configs` file (by default, stored at `~/.influxdbv2/configs`).
+
+## Usage
+```
+influx config create [flags]
+```
+
+## Flags{
+| Flag                | Description                                                    | Input type  |
+|:----                |:-----------                                                    |:----------: |
+| `-a,`, `--active`   | Set the specified connection to be the active configuration.   |             |
+| `-h`, `--help`      | Help for the `create` command                                  |             |
+| `-n`, `--name`      | (**Required**) Name of the new configuration.                  | string      |
+| `-o`, `--org`       | Organization name                                              | string      |
+| `u`, `--url`        | (**Required**) Connection URL for the new configuration.       | string      |
```

Comment:
-u here
also, missing -t --token which is also required

Reply:
@kelwang I fixed this on master. Also, `token` is a global flag, so it's listed separately.